src/App.tsx

```diff
@@ -1,57 +1,29 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { peopleFromServer } from './data/people';
+import { DropDown } from './components/DropDown';
+import { Person } from './types/Person';
```

Comment:
please fix this linter issue.
looks like it is because you didn't commit and push all files with changes.
![Screenshot from 2023-10-02 13-44-36](https://github.com/mate-academy/react_autocomplete/assets/33029864/b495f853-fc5e-4953-96a3-0124eeaaf60d)
![Screenshot from 2023-10-02 13-44-27](https://github.com/mate-academy/react_autocomplete/assets/33029864/3e66a729-f726-4d6b-9591-965704522bc0)